docs/detections/alerts-run-osquery.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[alerts-run-osquery]]
+=== Run Osquery
+Run live queries against the alert's host to learn more about your infrastructure and operating systems.
+
+IMPORTANT: The host associated with the alert must have the Osquery manager integration enabled. In addition, to use Osquery, your role requires specific privileges. Refer to {kibana-ref}/osquery.html[Osquery] for more information.
+
+. Click the *View details* button from the Alerts table to open the Alert details flyout.
+. Click *Take action*, then select *Run Osquery*.
+. Select one or more agents or groups to query. Start typing in the search field to get suggestions for agents by name, ID, platform, and policy.
+
++
+NOTE: The host associated with the alert is automatically selected. You can specify additional hosts to query.
++
+
+. Enter a query or select a query from your saved queries.
+. (Optional) Expand the **Advanced** section to view or set {kibana-ref}/advanced-osquery.html[mapped ECS fields] included in the results from the live query.
+. Click **Submit**.
+
++
+TIP: To save the query for future use, click *Save for later* and define the ID,
+description, and other {kibana-ref}/osquery.html#osquery-manage-query[details].
++
+
+[role="screenshot"]
+image::images/setup-query.png[width=80%][height=80%][Shows how to set up the query]
+
+. Review the results in a table, or navigate to *Discover* to dive deeper into the response,
+or to the drag-and-drop *Lens* editor to create visualizations.
+. To view more information about the request, such as failures, open the *Status* tab in the results table.
+
+[role="screenshot"]
+image::images/query-results.png[width=80%][height=80%][Shows query results]
```

Comment:
I think adding `+` in the blank line will scootch the image in line with the previous step (like the previous image is aligned), instead of the overall left margin:
```suggestion
. To view more information about the request, such as failures, open the *Status* tab in the results table.
+
[role="screenshot"]
image::images/query-results.png[width=80%][height=80%][Shows query results]
```